keps/prod-readiness/sig-instrumentation/3466.yaml

```diff
@@ -6,3 +6,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
Few questions:

1. From beta criteria:

> Gather feedback from developers
> Feature implemented for other Kubernetes Components

Did you gather the feedback? What components implement it now? Can you extend the KEP to update this?


> Are there any tests for feature enablement/disablement?

You have this question twice with two different answers :)
According to template, please only leave the second one, but please copy the answer from the first one.

Reply:
removed one of the repeated questions and fixed some of the verbage in the KEP.

Reply:
```
wojtekt@wojtekt-l:~/go/src/k8s.io/kubernetes$ git grep "ComponentSLIs" | grep -v CHANGELOG
staging/src/k8s.io/component-base/metrics/features/kube_features.go:	ComponentSLIs featuregate.Feature = "ComponentSLIs"
staging/src/k8s.io/component-base/metrics/features/kube_features.go:		ComponentSLIs: {Default: true, PreRelease: featuregate.Beta},
```

I don't see any components relying on the FG...
Are all of these implement the SLIs? Why isn't it gated?

Reply:
https://github.com/kubernetes/kubernetes/pull/120574 shipped too early. My mistake, I forgot to put a hold on it.

Reply:
OK - thanks.

Please just split "controller-manager" to both "cloud-controller-manager" and "kube-controller-manager"

and I'm fine with that then :)

Reply:
Done.

Reply:
Thanks!